common/app/model/Badges.scala

```diff
@@ -22,16 +20,8 @@ case class SpecialBadge(salt: String, hashedTag: String, imageUrl: String) exten
       Some(Badge(tag, imageUrl))
     } else None
 
-  private val digest = MessageDigest.getInstance("MD5")
-
   private def md5(input: String): Option[String] = {
-    try {
-      digest.update(input.getBytes(), 0, input.length)
-
-      Option(new BigInteger(1, digest.digest()).toString(16))
-    } catch {
-      case NonFatal(_) => None
-    }
+    Some(DigestUtils.md5Hex(input))
```

Comment:
```suggestion
    DigestUtils.md5Hex(input)
```

Maybe? Or are you just leaving this as optional so it can be reverted later?